tabs/failsafe.html

```diff
@@ -10,25 +10,7 @@
                 </p>
             </div>
         </div>
-        <div class="note oldpane">
```

Comment:
Check this, https://github.com/betaflight/betaflight-configurator/blob/master/tabs/failsafe.js#L67-L70
For apiVersions greater or equal then 1.15 then oldpane is just disabled and hidden, since we only should support 1.16 and later there is no need to have this just to disable it at one and never show it.